src/software/jetson_nano/thunderloop.cpp

```diff
@@ -56,6 +56,13 @@ Thunderloop::Thunderloop(const RobotConstants_t& robot_constants, const int loop
               << ", and NETWORK INTERFACE: " << network_interface_;
     LOG(INFO)
         << "THUNDERLOOP: to update Thunderloop configuration, change REDIS store and restart Thunderloop";
+
+    // Initializing default values for network and chipper kicker statuses
+    network_status_.set_ms_since_last_primitive_received(UINT32_MAX);
+    network_status_.set_ms_since_last_vision_received(UINT32_MAX);
+
+    chipper_kicker_status_.set_ms_since_chipper_fired(UINT32_MAX);
+    chipper_kicker_status_.set_ms_since_kicker_fired(UINT32_MAX);
```

Comment:
It doesn't make sense any more to initialize these fields like this since they're doubles